release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1024,6 +1024,15 @@ With this release, Operator authors can configure their Operator projects with s
 
 For more information, see xref:../operators/operator_sdk/osdk-multi-arch-support.adoc#osdk-multi-arch[Configuring Operator projects for multi-platform support].
 
+[id="ocp-4.14-builds"]
+=== Builds
+
+The Source-to-Image (S2I) tool is now generally available in {product-title} {product-version}. With this update, you can use the S2I tool to build container images from source code, and transform application code into ready-to-deploy container images. This feature enhances the platform's ability to support reproducible containerized application development. 
+
```

Comment:
Do the entries for S2I and BuildCSIVolume each require there own heading? Asking, as these seem to be two distinct features being released.